### PR TITLE
Plugin UI window on place open & toolbar button fixes.

### PR DESCRIPTION
**Pull request by nestenes**
This fix prevents the plugin UI window from opening up when a studio place opens. 
Furthermore, this also fixes the case where the user would have to double click the toolbar button to click the UI.

### CODE REVIEW

File: src/init.server.lua [hunk 1]
```diff
@@ -167,7 +169,7 @@ if not IS_DEBUG_MODE then
 	screenGui:GetPropertyChangedSignal("Enabled"):Connect(function()
 		button:SetActive(screenGui.Enabled)
 	end)
-	button:SetActive(screenGui.Enabled)
+	--button:SetActive(screenGui.Enabled)
```
Comment (Quenty) [inline src/init.server.lua]: Just delete this line!

**Review by Quenty — CHANGES_REQUESTED** [review]
Looks great overall, just a request to delete commented out line!

**Comment by nestenes** [comment]
Line should be removed.

File: src/init.server.lua (hunk 1 above)
Comment (nestenes) [reply src/init.server.lua]: Line should be removed.

**Review by Quenty — APPROVED** [review]

**Pull request merged**